src/pages/Overview/index.js

```diff
@@ -1,16 +1,880 @@
+                            style={{ float: 'right' }}
+                          >
+                            View in Kibana
+                          </Button>
+                        </Text>
+                      </TextContent>
+                    </div>
+                  </Card>
+                  <Card className={styles.borderedCard}>
+                    <Grid>
+                      <GridItem span={8}>
+                        <TextContent className={styles.paddingSmall}>
+                          <Text component={TextVariants.h4} style={{ color: 'grey' }}>
+                            {' '}
+                            @metadata.satellite.EC2: filters
+                          </Text>
+                        </TextContent>
+                        <Grid>
+                          <div className={styles.borderedCard}>
+                            <Chart
+                              ariaDesc="mock metdata"
+                              ariaTitle="mock meta data"
+                              containerComponent={
+                                <ChartVoronoiContainer
+                                  labels={({ datum }) => `${datum.name}: ${datum.y}`}
+                                  constrainToVisibleArea
+                                />
+                              }
+                              domainPadding={{ x: [50, 50] }}
+                              height={500}
+                              padding={{
+                                bottom: 50,
+                                left: 50,
+                                right: 200,
+                                top: 50,
+                              }}
+                              width={1200}
+                            >
+                              <ChartAxis />
+                              <ChartAxis dependentAxis showGrid />
+                              <ChartStack>
+                                <ChartBar
+                                  data={[
+                                    { name: 'meta1', x: '5-10-2020', y: 100 },
+                                    { name: 'meta1', x: '6-10-2020', y: 100 },
+                                    { name: 'meta1', x: '7-10-2020', y: 70 },
+                                    { name: 'meta1', x: '8-10-2020', y: 10 },
+                                    { name: 'meta2', x: '9-10-2020', y: 50 },
+                                    { name: 'meta2', x: '10-10-2020', y: 50 },
+                                    { name: 'meta2', x: '11-10-2020', y: 30 },
+                                    { name: 'meta3', x: '12-10-2020', y: 50 },
+                                    { name: 'meta2', x: '13-10-2020', y: 50 },
+                                    { name: 'meta2', x: '14-10-2020', y: 30 },
+                                    { name: 'meta3', x: '15-10-2020', y: 150 },
+                                    { name: 'meta2', x: '16-10-2020', y: 50 },
```

Comment:
Can we reference this as an object defined in a mock file or it wont work that way, I'm not sure about it?